author.hbs

```diff
@@ -1,33 +1,55 @@
 {{!< default}}
 {{! The tag above means - insert everything in this file into the {body} of the default.hbs template }}
 
+{{> "nav"}}
+
 {{! The big featured header }}
-<header class="main-header author-head {{#if author.cover}}" style="background-image: url({{author.cover}}){{else}}no-cover{{/if}}">
-    <nav class="main-nav overlay clearfix">
-        <a class="back-button icon-arrow-left" href="{{@blog.url}}">Home</a>
-        <a class="subscribe-button icon-feed" href="{{@blog.url}}/author/{{author.slug}}/rss/">{{author.name}}</a>
-    </nav>
+<header class="main-header no-cover">
 </header>
 
 <section class="author-profile inner">
+
     {{#if author.image}}
-    <figure class="author-image">
-        <div class="img" style="background-image: url({{author.image}})"><span class="hidden">{{author.name}}'s Picture</span></div>
-    </figure>
+
+    <div class="row">
+	    <figure class="small-8 columns author-image">
+	        <div class="img" style="background-image: url({{author.image}})">
+	            <span class="hidden">{{author.name}}'s Picture</span>
+	        </div>
+	    </figure>
+    </div>
+
     {{/if}}
+
     <h1 class="author-title">{{author.name}}</h1>
-    <h2 class="author-bio">{{author.bio}}</h2>
+
+	{{#if author.bio}}
+	    <div class="row">
+		    <p class="small-10 large-10 small-centered large-centered columns author-bio">
```

Comment:
Let's add a `text-left` class here, I think (especially on mobile) the bio text makes more sense when it is left aligned.